all/pom.xml

```diff
@@ -232,250 +249,395 @@
           <version>${project.version}</version>
           <classifier>${jni.classifier}</classifier>
           <scope>compile</scope>
+          <optional>true</optional>
         </dependency>
         <dependency>
           <groupId>${project.groupId}</groupId>
           <artifactId>netty-resolver-dns-native-macos</artifactId>
           <scope>compile</scope>
+          <optional>true</optional>
         </dependency>
         <!-- Just include the classes for the other platform so these are at least present in the netty-all artifact -->
         <dependency>
           <groupId>${project.groupId}</groupId>
           <artifactId>netty-transport-native-epoll</artifactId>
           <scope>compile</scope>
+          <optional>true</optional>
         </dependency>
       </dependencies>
     </profile>
-
-    <profile>
-      <id>full</id>
```

Comment:
This is not needed anymore as we moved the Javadoc etc generated to the parent pom.xml